integrations/gcp/.port/resources/blueprints.json

```diff
@@ -4,7 +4,12 @@
     "title": "GCP Project",
     "icon": "GoogleCloud",
     "schema": {
-      "properties": {}
+      "properties": {
+        "labels": {
```

Comment:
Aren't labels an array of type object?

Reply:
<img width="1055" alt="Screenshot 2024-07-18 at 10 24 51 AM" src="https://github.com/user-attachments/assets/71c22e7a-3912-4e3e-aa68-43e99ac0208d">

Labels in GCP are represented as objects instead of array